proto/forch_configuration.proto

```diff
@@ -0,0 +1,45 @@
+/*
+ * Encapsulating Forch configuration
+ */
+syntax = "proto3";
+
+// TODO fill up sections one by one
+/*
+ * Forch configuration
+ */
+message ForchConfig {
+  // configurations for processes
+  ProcessConfig process = 1;
+}
+
+/*
+ * Processes section in Forch configuration
+ */
+message ProcessConfig {
+  // scan interval in seconds
+  int32 scan_interval_sec = 1;
+
+  // indicate if Forch should check VRRP state
+  bool check_vrrp = 2;
+
+  // target processes to check
+  map<string, Process> processes = 3;
+
+  // connections to check
```

Comment:
for maps, should indicate what the key is (e.g., in this case it's "listening port")

Reply:
Added